pkg/plugins/manager/signature/manifest.go

```diff
@@ -342,11 +341,7 @@ func (s *Signature) Verify(ctx context.Context, keyID string, block *clearsign.B
 	if _, err = openpgp.CheckDetachedSignature(keyring,
 		bytes.NewBuffer(block.Bytes),
 		block.ArmoredSignature.Body, &packet.Config{}); err != nil {
-		// If the key includes revocations, we can assume that the key was revoked
-		if len(keyring) > 0 && len(keyring[0].Revocations) > 0 {
-			return fmt.Errorf("%s (KeyID: %s): %w", openpgpErrors.ErrKeyRevoked.Error(), keyID, err)
-		}
-		return fmt.Errorf("%v: %w", "failed to check signature", err)
+		return fmt.Errorf("%s: %w", "failed to check signature", err)
```

Comment:
Nit: I find this format easier to read instead:

```suggestion
		return fmt.Errorf("failed to check signature: %w", err)
```